File: backend-module/src/main/java/com/campus/eneity/TeachingTask.java
package com.campus.eneity;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class TeachingTask {
    private int id;
    private int teacherId;
    private int classId;
    private int courseId;
    private String semester;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private boolean isDeleted;
}

File: frontend/src/types/teachingTask.ts
export interface TeachingTask {
  id: number
  teacherId: number
  classId: number
  courseId: number
  semester: string
  createdAt: string
  updatedAt: string
  isDeleted: boolean
}

export interface TeachingTaskForm {
  id?: number
  teacherId: number | null
  classId: number | null
  courseId: number | null
  semester: string
}
